prototype/requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyramid
 requests
-
+nose
```

Comment:
Yeah, technically they should be but being small, and potentially unused, I decided to keep it simple.